java/runtime/src/main/java/io/ray/runtime/object/NativeObjectStore.java

```diff
@@ -44,6 +47,25 @@ public void delete(List<ObjectId> objectIds, boolean localOnly, boolean deleteCr
     nativeDelete(toBinaryList(objectIds), localOnly, deleteCreatingTasks);
   }
 
+  @Override
+  public void addLocalReference(UniqueId workerId, ObjectId objectId) {
+    nativeAddLocalReference(workerId.getBytes(), objectId.getBytes());
+  }
+
+  @Override
+  public void removeLocalReference(UniqueId workerId, ObjectId objectId) {
+    nativeRemoveLocalReference(workerId.getBytes(), objectId.getBytes());
+  }
+
+  public Map<ObjectId, long[]> getAllReferenceCounts() {
```

Comment:
What is this method used for?

Reply:
I assume this is to mirror the internal API we have in python?

Reply:
This method is only used in unit tests to check the ref counts.